pkg/cli/add_vcluster_helm.go

```diff
@@ -24,31 +27,79 @@ type AddVClusterOptions struct {
 	AccessKey                string
 	Host                     string
 	CertificateAuthorityData []byte
+	All                      bool
 }
 
 func AddVClusterHelm(
 	ctx context.Context,
 	options *AddVClusterOptions,
 	globalFlags *flags.GlobalFlags,
-	vClusterName string,
+	args []string,
 	log log.Logger,
 ) error {
-	// check if vCluster exists
-	vCluster, err := find.GetVCluster(ctx, globalFlags.Context, vClusterName, globalFlags.Namespace, log)
-	if err != nil {
-		return err
+	var vClusters []find.VCluster
+	if len(args) == 0 && !options.All {
+		return errors.New("empty vCluster name but no --all flag set, please either set vCluster name to add one cluster or set --all flag to add all of them")
```

Comment:
I personally wouldn't exit with an error here. I'd rather try to display the default help message instead

Reply:
This is probably a question of the CLI style, but I feel we should error here (as it is an user input error). 
For example, I can imagine someone using this CLI command in the CI job, in a script with `set -e` and passing vcluster name dynamically (e.g. from the env var). In the case when name is empty and we don't error here (and exit with `0`), then the pipeline may continue and user may expect that vcluster got added to platform.